packages/contracts-ops/contracts/test/Reboot.t.sol

```diff
@@ -4,13 +4,33 @@ pragma abicoder v2;
 
 import "forge-std/Test.sol";
 import {RebootLogic} from "../scripts/Reboot.s.sol";
-import {ReplicaTest} from "@nomad-xyz/contracts-core/contracts/test/Replica.t.sol";
+import {NomadTest, NomadTestWithUpdaterManager, ReplicaHandlers} from "@nomad-xyz/contracts-core/contracts/test/utils/NomadTest.sol";
 import {ReplicaHarness} from "@nomad-xyz/contracts-core/contracts/test/harnesses/ReplicaHarness.sol";
+import {ReplicaTest, ReplicaHandlers} from "@nomad-xyz/contracts-core/contracts/test/Replica.t.sol";
+import {HomeTest} from "@nomad-xyz/contracts-core/contracts/test/Home.t.sol";
+import {HomeHarness} from "@nomad-xyz/contracts-core/contracts/test/harnesses/HomeHarness.sol";
 
-contract RebootTest is RebootLogic, ReplicaTest {
+contract RebootTest is
+    RebootLogic,
+    NomadTest,
+    NomadTestWithUpdaterManager,
+    ReplicaHandlers,
+    ReplicaTest,
+    HomeTest
+{
     string remote;
 
-    function setUp() public override {
+    function setUp()
+        public
+        override(
+            NomadTest,
+            NomadTestWithUpdaterManager,
+            ReplicaHandlers,
+            ReplicaTest,
+            HomeTest
+        )
+    {
+        vm.createSelectFork(vm.envString("RPC_URL"), 15_977_625);
```

Comment:
how much of this will need to be at the start of every fork test and can be moved to a fork test base contract?

Reply:
ack, thinking about refactor as soon as foundry is updated to fix race condition